grafana-plugin/src/pages/outgoing_webhooks_2/OutgoingWebhooks2.tsx

```diff
@@ -264,20 +263,27 @@ class OutgoingWebhooks2 extends React.Component<OutgoingWebhooks2Props, Outgoing
           />
         </CopyToClipboard>
         <WithPermissionControlTooltip key={'status_action'} userAction={UserActions.OutgoingWebhooksRead}>
-          <Button onClick={() => this.onStatusClick(record.id)} fill="text">
-            Status
-          </Button>
+          <IconButton
+            tooltip="Status"
+            tooltipPlacement="top"
+            name="history"
+            onClick={() => this.onStatusClick(record.id)}
+          />
         </WithPermissionControlTooltip>
         <WithPermissionControlTooltip key={'edit_action'} userAction={UserActions.OutgoingWebhooksWrite}>
-          <Button onClick={() => this.onEditClick(record.id)} fill="text">
-            Edit
-          </Button>
+          <IconButton tooltip="Edit" tooltipPlacement="top" name="cog" onClick={() => this.onEditClick(record.id)} />
+        </WithPermissionControlTooltip>
+        <WithPermissionControlTooltip key={'copy_action'} userAction={UserActions.OutgoingWebhooksWrite}>
+          <IconButton tooltip="Copy" tooltipPlacement="top" name="copy" onClick={() => this.onCopyClick(record.id)} />
```

Comment:
Maybe use `Duplicate` instead? Because we have used Copy already for clipboard, this is more like a Duplicate functionality. Up to you.